server/service/src/stock_line/update.rs

```diff
@@ -279,21 +279,21 @@ fn log_stock_changes(
         )?;
     }
     if existing.on_hold != new.on_hold && new.on_hold {
-        activity_log_stock_entry(
+        activity_log_entry(
             &ctx,
             ActivityLogType::StockOnHold,
             Some(new.id.to_owned()),
-            Some("off hold".to_string()),
-            Some("on hold".to_string()),
+            Some("\u{2705}".to_string()), // tick unicode
```

Comment:
Wasn't too sure what kind of unicode to use or which way around they should be... even emojis can be country-specific! 😭